client/src/containers/Authentication.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Menu } from "semantic-ui-react";
+import SigninForm from "./SigninForm";
+import SignupForm from "./SignupForm"
+
+const Authentication = () => {
+  const [activeItem, setActiveItem] = useState("signin");
+
+  const renderForm = () => {
+    if (activeItem === "signin") {
+      return (
+        <SigninForm/>
+      );
+    } else {
+      return (
+        <SignupForm/>
+      );
```

Comment:
The signup form will render every time the activeItem contains a different value  than `signin`.